src/applications/ivc-champva/10-10D/pages/ApplicantMedicareStatusContinuedPage.jsx

```diff
@@ -11,7 +11,9 @@ const keyname = 'applicantMedicarePart';
 function generateOptions({ data, pagePerItemIndex }) {
   const applicant = applicantWording(
     data?.applicants?.[pagePerItemIndex],
-  ).slice(0, -3); // remove 's_
+    undefined,
+    false,
+  );
```

Comment:
Updating the usage of this recently refactored `applicantWording` function. Not directly related to the issue at hand, but was causing strings to be truncated on other pages so I fixed it here and in two other places.